client/py/synnax/framer/writer.py

```diff
@@ -102,6 +104,9 @@ class Writer:
     error, close will raise the accumulated error.
     """
 
+    # constant
+    ALWAYS_INDEX_PERSIST_ON_AUTO_COMMIT: TimeSpan = -1
```

Comment:
Why is this a static variable on the request class? Shouldn't you move it to the module scope?

Reply:
@LeonLiur You never addressed this

Reply:
Moved to module scope